### PR TITLE
[pt-br] Translate glossary tag display names

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes

* Add translations for glossary tag display names in Brazilian Portuguese, as enabled through #37966.

/language pt
/cc @edsoncelio @MrErlison @rikatz

### CODE REVIEW

**Review by MrErlison — APPROVED** [review]
Thank you!

**Comment by edsoncelio** [comment]
Thanks!
/lgtm

File: data/i18n/pt/pt-br.toml [hunk 1]
```diff
@@ -238,3 +238,75 @@ other = "Retornar à visualização normal"
...
+other = "Armazenamento"
+
+[layout_docs_glossary_storage_description]
+other = "Como as aplicações que rodam no Kubernetes manipulam dados persistentes."
+
+[layout_docs_glossary_tool_name]
+other = "Ferramentas"
+
+[layout_docs_glossary_tool_description]
+other = "Software que torna o Kubernetes mais fácil ou melhor de ser utilizado."
```
Comment (devlware) [inline data/i18n/pt/pt-br.toml:300]: nit: How about:
"Software que torna o Kubernetes mais fácil ou seu uso melhor." ?

**Review by devlware — COMMENTED** [review]
Thank you @stormqueen1990.

**Pull request merged**